components/work-schedule/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styles from "../work-schedule/work-schedule.module.css";
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+} from "@material-ui/core/";
+import AlarmRoundedIcon from "@material-ui/icons/AlarmRounded";
+import HourglassEmptyRoundedIcon from "@material-ui/icons/HourglassEmptyRounded";
+import HourglassFullRoundedIcon from "@material-ui/icons/HourglassFullRounded";
+import { red } from "@material-ui/core/colors";
+
+export interface WorkScheduleProp {
+  workDay: string;
+  startTime: string;
+  endTime: string;
+}
+
+const WorkSchedule: React.FunctionComponent<WorkScheduleProp> = (props) => {
+  return (
+    <div className={styles.div}>
+      <List className={styles.list}>
+        <ListItem className={styles.listItem}>
+          <ListItemAvatar>
+            <Avatar>
+              <AlarmRoundedIcon style={{ color: red[500] }} />
```

Comment:
Do you want to make this white? I think red is too contrasting with all the colors that we have.